### DISCUSSION TITLE
Does Solito support fullstack development via next.js pages/api ?

### DISCUSSION THREAD

**Discussion by asalaza6** [nandorojo/solito#166] [General]
Does Solito support fullstack development via next.js pages/api ? Can I take advantage of this next.js feature?

**Comment by karmatradeDev** [comment]
yes, I use this in my application the yarn workspace allows for the separation of native apps and nextjs.  I use vercel to host and it runs next build inside the apps/next folder.

this may be weird but I had to configure it like this to allow for modules inside of package/app to install 
<img width="939" alt="Screen Shot 2022-09-28 at 3 46 50 PM" src="https://user-images.githubusercontent.com/86848104/192885497-0802b082-36fb-44f9-9dd2-b39036603f18.png">

**Reply by nandorojo** [reply]
hmm that shouldn’t be the case. vercel should as you your root directory and that’s it

**Reply by karmatradeDev** [reply]
i might have done something weird ill look into removing the override and see what happens. thanks

**Reply by asalaza6** [reply]
Okay nice, thanks for the comment. So I can use the fullstack api, host the web fe and be on vercel, and the mobile version will use the nextjs vercel api right?

**Reply by nandorojo** [reply]
yeah you’ll just need to set a different endpoint on native

**Reply by gascenciom1998** [reply]
@nandorojo can you point me to the docs/example on how to set up an endpoint on native? I'm still a bit lost as to how the expo app can access the pages/api endpoints
